### DISCUSSION TITLE
Why Cat5 cable? There's a story here ...

### DISCUSSION THREAD

**Discussion by peterantypas** [peterantypas/maiana#84] [General]
When I originally designed MAIANA&trade; and installed it on my boat, the interface between the transponder unit and the adapter in the cabin was a 4-wire cable: It had 12V, GND, UART TX and UART RX and nothing else. 

On the transponder side, it was attached to the board using a 0.1" pitch screw terminal (Phoenix type).  This made assembly a bit challenging: The cable had to be attached to the board first, then the board had to be inserted into the antenna enclosure and sealed. Finally, the cable would need to be run through the antenna mast, to the cabin, and then pulled all the way through leaving just enough slack for the antenna enclosure to sit onto its mast (pipe). Disassembly of the unit required the reverse operation and it was just painful.

So I experimented with a 4-wire phone cable instead (RJ11). It seemed like a logical choice. The cable could be routed once, and then simply snapped in and out of jacks on the opposite ends. It was the year 2018 though, and with landlines nearing extinction globally, I figured relying on phone cable wasn't the best choice. In addition, I wanted to add a "TX off" (hardware) switch to the control unit in the cabin, and that would need one more signal. So that's what prompted me to go with 8-wire Cat5 (Ethernet) cable instead. The addition of RX/TX/GPS status LED signals was more of an afterthought ("what do I do with these extra 3 wires now?")

That said, I recently had a situation which was by far the most challenging of the 30+ installs so far. The system would not run normally, would not acquire a GPS fix, would not output NMEA data, and the power LED would occasionally flash. My immediate reaction was to send replacement boards to my customer who was preparing for an Atlantic passage to the Canary Islands. But after much angst and an expensive DHL shipment to Spain, the replacement boards failed just as spectacularly as the original ones :(

And then it dawned on me: All the problems he was having could be attributed to a bad Cat5 cable. Many strange things that can happen if any of the wires do not make contact or have the wrong kind of short between them. And sure enough, that was the root cause!

I still think that the use of Cat5 was a reasonable design decision, but I'm not religious about it anymore. As I'm rethinking the HW design with an eye toward tight OpenPlotter integration, I'm willing to consider alternatives, especially if the alternative is easy to supply as part of the kit and then it's a known quantity. The main requirement is that it carries at least 8 signals and it's really easy to disconnect from the transponder side when servicing.

Any and all suggestions welcome!

**Comment by Peter-Petrik** [comment]
From my installation experience, CAT5/6 cable is a great solution. It is widely available in many different pre-cut lengths, can be easily crimped to custom length, a number of outdoor/exterior grades are available, and most of all, lots of inexpensive testers that confirm the cable isn't cut, kinked with a bad connection, or crimped incorrectly. All of that made it a great solution for my installation even though I needed to source it myself.

**Reply by muebau** [reply]
Yes I agree. It was easy and cheap to get a UV resistant version. I wonder if a different type of cabling would come even near the price, availability, good properties and quality of a CAT6 cable. For my part I was very happy with the design decision.
